packages/config/src/VitessceAutoConfig.js

```diff
@@ -21,13 +40,14 @@ class OmeTiffAutoConfig extends AbstractAutoConfig {
     this.fileName = fileUrl.split('/').at(-1);
   }
 
-
-  async composeViewsConfig() { /* eslint-disable-line class-methods-use-this */
-    return [
-      ['description'],
-      ['spatial'],
-      ['layerController'],
+  async composeViewsConfig(hintsConfig) { /* eslint-disable-line class-methods-use-this */
+    const possibleViews = [
+      ['description'], ['spatial'], ['layerController'],
     ];
+    if (Object.keys(hintsConfig).length === 0) {
+      return possibleViews;
+    }
+    return filterViews(possibleViews, Object.keys(hintsConfig.views));
```

Comment:
It seems like this code block is repeated before every call to `filterViews`, can it be turned into a function?